app/Controllers/ControllerInterface.php

```diff
@@ -0,0 +1,13 @@
+<?php
+
+declare(strict_types=1);
+
+namespace App\Controllers;
+
+use App\Entities\Response;
+
+interface ControllerInterface
+{
+    public function getAll(): Response;
+    public function getById(int $id): Response;
```

Comment:
I wouldn't enforce controllers to have these methods but I think it is okay, not big deal so no changes are needed.